FastBank.ConsoleApplication/MenuOptions.cs

```diff
@@ -58,6 +61,11 @@ static public void ShowMainMenu()
                     RenderMenuByRole();
                 }
             }
+
+            Console.Clear();
+            Console.WriteLine("You successfully exited the system! Pres any key to close programm...");
```

Comment:
Pres -> Press
Pay attention and read everything before commit.

Reply:
done